### PR TITLE
[Jonathan Iheme]-Sage-Build A Full Stack Decentralized Fund Raising Dapp On The Celo Blockchain

### PR DESCRIPTION
**Pull request by 4undRaiser**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/03-08-2023-Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain/Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain.md [hunk 1]
```diff
@@ -0,0 +1,666 @@
...
+
+- Clone the repo to your computer.
+- open the project from from vscode.
+- Run `npm install` command to install all the dependencies required to run the app locally.
+
+#### App.js
+
+The completed code should look like this.
+
+```solidity
```
Comment (Julius170) [inline blog/03-08-2023-Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain/Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain.md:344]: this should be javascript

File: blog/03-08-2023-Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain/Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain.md [hunk 2]
```diff
@@ -0,0 +1,666 @@
...
+
+I hope you learned a lot from this tutorial. Here are some relevant links that would aid your learning further.
+
+- <https://docs.celo.org/>
+- <https://solidity-by-example.org/>
+- <https://docs.soliditylang.org/en/v0.8.17/>
+
+## About the author
+
+I'm Jonathan Iheme, A full stack block-chain Developer from Nigeria.
```
Comment (Julius170) [inline blog/03-08-2023-Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain/Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain.md:664]: not compulsory but why not add a link to you linkedIn Page, helps with the traffic
something like, You can connect with me on [LinkedIn](www.you.link.here)

File: blog/03-08-2023-Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain/Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain.md [hunk 3]
```diff
@@ -0,0 +1,666 @@
...
+export default App;
+
+```
+And finally, we render the App component and return the Home and Campaigns components with the necessary props.
+
+## Next Steps
+
+I hope you learned a lot from this tutorial. Here are some relevant links that would aid your learning further.
+
+- <https://docs.celo.org/>
```
Comment (Julius170) [inline blog/03-08-2023-Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain/Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain.md:658]: can you embed the links here like you've done throughout the tutorial 🙃

**Review by Julius170 — COMMENTED** [review]
Hi @4undRaiser your piece looks good Great Job, Just a few corrections and you're all set for publish

File: blog/03-08-2023-Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain/Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain.md (hunk 3 above)
Comment (4undRaiser) [reply blog/03-08-2023-Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain/Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain.md:658]: it's important for readers to see the links they are going to if they want to learn more about a specific topic

Comment (Julius170) [reply blog/03-08-2023-Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain/Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain.md:658]: No seeing the link is not what's important, taking the reader to the right reference is what is
That is why you should embed it with the right description or example:

the first link would rather look like this
[celo docs](https://docs.celo.org/)

File: blog/03-08-2023-Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain/Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain.md [hunk 4]
```diff
@@ -0,0 +1,666 @@
...
+- <https://solidity-by-example.org/>
+- <https://docs.soliditylang.org/en/v0.8.17/>
+
+## About the author
+
+I'm Jonathan Iheme, A full stack block-chain Developer from Nigeria. let's connect on (linkedIn)[https://www.linkedin.com/in/jonathan-iheme-31a63718b]
```
Comment (Julius170) [inline blog/03-08-2023-Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain/Build-A-Full-Stack-Decentralized-Fund-Raising-Dapp-On-The-Celo-Blockchain.md:664]: The link you have here is also not embedded proprely 
there is a space between "linkedin" and the link,

**Review by Julius170 — COMMENTED** [review]
Still haven't corrected the changes @4undRaiser

**Review by developerkunal — APPROVED** [review]
Approved @Ishanpathak1

**Pull request merged**